### PR TITLE
Simplify example code

### PR DESCRIPTION
**Pull request by michalciurus**
Simplifies the example project code so users can focus on Blockstack code, rather than UI code.

### CODE REVIEW

File: Example/Blockstack/ViewController.swift [hunk 1]
```diff
@@ -12,23 +12,8 @@ class ViewController: UIViewController {
...
     @IBOutlet var nameLabel: UILabel?
-    
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        
-        signInButton?.setTitle("Sign In with Blockstack", for: .normal)
-        signInButton?.setTitleColor(.white, for: .normal)
-        signInButton?.setTitleColor(.white, for: .highlighted)
-        signInButton?.backgroundColor = UIColor.black
-        signInButton?.addTarget(self, action: #selector(signin), for: UIControlEvents.touchUpInside)
```
Comment (michalciurus) [inline Example/Blockstack/ViewController.swift:23]: Moved to storyboard so users can focus on Blockstack code

File: Example/Blockstack/ViewController.swift [hunk 2]
```diff
@@ -52,24 +36,22 @@ class ViewController: UIViewController {
         let retrievedUserData = Blockstack.sharedInstance().loadUserData()
         print(retrievedUserData?.profile?.name as Any)
         
-        DispatchQueue.main.async {
-            let name: String? = retrievedUserData?.profile?.name
-            self.nameLabel?.text = "Hello, \(name!)"
-            self.signInButton?.isHidden = true
-        }
```
Comment (michalciurus) [inline Example/Blockstack/ViewController.swift]: I'm pretty sure`SFAuthenticationSession` runs its completion on main queue so there's no need to additionallly run UI code in a main queue closure. The documentation doesn't specify it explicitly, but I tested it myself plus found some [example code](https://github.com/dvdhpkns/SFAuthenticationSession-example/blob/master/SFAuthenticationSessionExample/SFAuthenticationSessionExample/ViewController.swift) that seems to prove it.

It's much safer and simpler if all Blockstack completion blocks *always* run on main queue - SDK users will avoid bugs this way and won't have to wonder on what queue does the completion run.

**Comment by yknl** [comment]
looks good to me 👍 merging this

**Comment by yknl** [comment]
Just tested this again and seems like the completion handler wasn't called on the main queue. Did this test on the simulator. Going to revert the removal of using the main thread for now.

```
Main Thread Checker: UI API called on a background thread: -[UILabel setText:]
PID: 77923, TID: 10851804, Thread name: (none), Queue name: NSOperationQueue 0x60400022f1e0 (QOS: UNSPECIFIED), QoS: 0
Backtrace:
4   Blockstack_Example                  0x000000010d6427d7 _T018Blockstack_Example14ViewControllerC19handleSignInSuccessy0A07PayloadV8userData_tF + 9063
```

**Comment by michalciurus** [comment]
@yknl 

Sorry, I just checked cancel/failure case. Didn't check the success case!

In case of success case `fetch` function is called from `ProfileHelper`. It uses `dataTask` from `URLSession` **which does run on a background thread!** - you're right!

So the Blockstack API method `signIn` calls the completion sometimes on the main queue, sometimes on the background queue :D I'll create an issue for this.

**Pull request merged**